adapter/service/chargepoint/routing.go

```diff
@@ -214,6 +217,44 @@ func handleCmdCableLockGetReport(serviceRegistry adapter.ServiceRegistry) router
 	)
 }
 
+// routeCmdCurrentSessionSetCurrent returns a routing responsible for handling the command.
+func routeCmdCurrentSessionSetCurrent(serviceRegistry adapter.ServiceRegistry) *router.Routing {
+	return router.NewRouting(
+		handleCmdCurrentSessionSetCurrent(serviceRegistry),
+		router.ForService(Chargepoint),
+		router.ForType(CmdCurrentSessionSetCurrent),
+	)
+}
+
+// handleCmdCurrentSessionSetCurrent returns a handler responsible for handling the command.
+func handleCmdCurrentSessionSetCurrent(serviceRegistry adapter.ServiceRegistry) router.MessageHandler {
+	return router.NewMessageHandler(
+		router.MessageProcessorFn(func(message *fimpgo.Message) (*fimpgo.FimpMessage, error) {
+			chargepoint, err := getService(serviceRegistry, message)
+			if err != nil {
+				return nil, err
+			}
+
+			current, err := message.Payload.GetIntValue()
```

Comment:
Is current going to be always an integer?

Reply:
Yes.